### DISCUSSION TITLE
Mouse hover over a contributor shows pop up dialog that *never* goes away on just normal GitHub web page

### DISCUSSION THREAD

**Discussion by imikejackson** [community/community#44812] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Lately I have noticed that on my web browsers (FireFox, Edge, Chrome, Safari) that when my mouse hovers over a contributor to a project then the normal pop up appears showing me some more detailed information about that contributor. My issue is that the pop up NEVER goes away. I can scroll the web page and come back and the pop up is still there. This is on several macOS systems from Catalina to Monterey with all 4 web browsers. Maybe I timer is missing on these platforms?

**Comment by radar-blip** [comment]
Similar discussion here: #44630

Selected answer suggests the team is aware of the problem.
